CMakeLists.txt

```diff
@@ -96,13 +101,7 @@ aws_use_package(aws-c-auth REQUIRED)
 aws_use_package(aws-checksums REQUIRED)
 target_link_libraries(${PROJECT_NAME} PRIVATE ${CMAKE_JS_LIB} ${DEP_AWS_LIBS})
 
-if (CMAKE_JS_PLATFORM AND CMAKE_JS_ARCH)
-    # When run from build.ts, these are set
-    set(destination bin/${CMAKE_JS_PLATFORM}-${CMAKE_JS_ARCH})
-else()
-    # If not, default to "native"
-    set(destination bin/native)
-endif()
+set(destination bin/${CMAKE_JS_PLATFORM}-${NODE_ARCH})
```

Comment:
Why we use `NODE_ARCH` instead?

Reply:
NODE_ARCH is a variable that is always set by cmake-js

Previously, we'd been using a variable we invented named `CMAKE_JS_ARCH`. There didn't seem to be a need for inventing our own variable, so I stopped using it

Reply:
So, `NODE_ARCH` comes from the `options.arch`?

Reply:
yeah, passing `options.arch` cmake-js leads to it setting this variable.

There are a few other variables from cmake-js that we're already using in our `CMakeLists.txt`, like: `CMAKE_JS_LIB` and `CMAKE_JS_INC`. I don't know whey they named this one `NODE_ARCH` instead of `CMAKE_JS_ARCH`, the naming is inconsistent. And I don't know why we chose to name a variable that we invented `CMAKE_JS_ARCH`, it makes it look like that variable is coming from cmake-js but it's not.

oh well. simpler now